iModelCore/ECPresentation/Source/Content/ContentQueryContracts.h

```diff
@@ -25,6 +26,8 @@ struct ContentQueryContract : PresentationQueryContract
     ECClassCP m_relationshipClass;
     Utf8String m_relationshipClassAlias;
     IQueryInfoProvider const& m_queryInfo;
+    ECSchemaHelper const& m_schemaHelper;
+    IRulesPreprocessor& m_rulesPreprocessor;

```

Comment:
What do you think about passing a lambda to get label override specs rather than schema helper and rules preprocessor? IMO, these two have way more than we need and I'd like contracts to have as little dependencies as possible.

Reply:
Changed to `std::function`